### PR TITLE
Added Registry Editor Methods to Design and Desktop

### PR DESCRIPTION
**Pull request by maxcapodi78**

### CODE REVIEW

File: pyaedt/desktop.py [hunk 1]
```diff
@@ -691,6 +624,113 @@ def disable_autosave(self):
         """
         self._main.oDesktop.EnableAutoSave(False)
 
+    def change_license_type(self, license_type="Pool"):
+        """Change the License Type between ``Pack`` and ``Pool``.
+
+        ..note: The command returns True even if the Key is wrong due to API limitation.
```
Comment (akaszynski) [inline pyaedt/desktop.py:630]: This won't get rendered unless it follows this format
```suggestion
        .. note::
           The command returns True even if the Key is wrong due to API limitation.
```

File: pyaedt/desktop.py [hunk 2]
```diff
@@ -691,6 +624,113 @@ def disable_autosave(self):
...
+
+        Parameters
+        ----------
+        license_type : str, optional
+            Set License Type between ``Pack`` and ``Pool``.
```
Comment (akaszynski) [inline pyaedt/desktop.py:635]:
```suggestion
            Set license type.  Must be either ``"Pack"`` or ``"Pool"``.
```

File: pyaedt/desktop.py [hunk 3]
```diff
@@ -691,6 +624,113 @@ def disable_autosave(self):
...
+
+        Returns
+        -------
+        bool
```
Comment (akaszynski) [inline pyaedt/desktop.py:639]:
```suggestion
        bool
           ``True`` when successful, ``False`` when failed.
```

File: pyaedt/desktop.py [hunk 4]
```diff
@@ -691,6 +624,113 @@ def disable_autosave(self):
...
+        """
+        try:
+            self._main.oDesktop.SetRegistryString("Desktop/Settings/ProjectOptions/HPCLicenseType", license_type)
+            return True
+        except:
```
Comment (akaszynski) [inline pyaedt/desktop.py:644]: We should log failure here.

Comment (maxcapodi78) [reply pyaedt/desktop.py:644]: the API doesn't return always error unfortunately

**Pull request merged**